src/components/NetworkNodeSelector/NetworkNodeSelectorTs.ts

```diff
@@ -134,6 +133,7 @@ export class NetworkNodeSelectorTs extends Vue {
         } catch (error) {
             this.$store.dispatch('notification/ADD_ERROR', NotificationType.INVALID_NODE);
             console.log(error);
+            this.filteredData = this.customNodeData;
```

Comment:
not sure why this was added, I can't see it use in anywhere

Reply:
when a node doesn't have a public key, and harvesting tries to use the currently selected node by account for harvesting after failing to fetch publicKey the list disappears, so this just refreshing the `filteredData`.